examples/vue-example-app/src/router/index.js

```diff
@@ -0,0 +1,80 @@
+import Vue from 'vue'
+import VueRouter from 'vue-router'
+import Home from '../views/Home.vue'
+
+Vue.use(VueRouter)
+
+const routes = [
+  {
+    path: '/',
+    name: 'Home',
+    component: Home
```

Comment:
Should the Demo be the root route `/`? Do we still need this as Home or should we call this MapLayout.vue?

Reply:
according to the JIRA story, we should preserve the root route as is and place the examples under /demo

Reply:
makes sense.